test-suite/Makefile

```diff
@@ -131,9 +131,10 @@ bugs: $(BUGS)
 
 clean:
 	rm -f trace .nia.cache .lia.cache output/MExtraction.out
+	rm -f vos/Makefile vos/Makefile.conf
 	$(SHOW) 'RM        <**/*.stamp> <**/*.vo> <**/*.vio> <**/*.log> <**/*.glob>'
```

Comment:
We should update this

Alternatively we could stop using SHOW/HIDE since the command isn't especially noisy (unlike ocamlc commands which have tons of includes for instance)